package.json

```diff
@@ -39,7 +39,7 @@
     "pkg:mac": "pkg bin/bit.js --targets node10-macos-x64 --out-path releases/mac --options --no-warnings --config package.json",
     "pkg:windows": "pkg bin/bit.js --targets node10-win-x64 --out-path releases/windows --options --no-warnings --config package.json",
     "pkg:all": "pkg bin/bit.js --targets node10-macos-x64,node10-win-x64,node10-linux-x64 --out-path releases/ --options --no-warnings --config package.json",
-    "watch": "babel --watch src -d dist --verbose --source-maps --extensions \".ts,.tsx,.js\" --copy-files",
+    "watch": "babel --watch src -d dist --verbose --source-maps --extensions '.ts,.tsx,.js' --copy-files",
```

Comment:
@ranm8 if you change this to be with `'` instead of `\"` it won't work for windows.